packages/kbn-es/src/constants.js

```diff
@@ -0,0 +1,24 @@
+/*
+ * Licensed to Elasticsearch B.V. under one or more contributor
+ * license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright
+ * ownership. Elasticsearch B.V. licenses this file to you under
+ * the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+const path = require('path');
+
+exports.caCertPath = path.resolve(__dirname, '../../../test/dev_certs/ca.crt');
```

Comment:
Lets move these into `@kbn/dev-utils`, certs and paths, then import the paths from the places they're used. Make sure to add `@kbn/dev-utils` to package.json files where necessary. Also, can we use `CONST_FORMATTING`?